mlir/lib/Dialect/MIOpen/Transforms/LowerMIOpenOps.cpp

```diff
@@ -81,6 +83,8 @@ void LowerMIOpenOpsStep3Pass::runOnOperation() {
   patterns.insert<BlockwiseGemmRewritePattern>(&getContext());
   patterns.insert<BlockwiseCopyRewritePattern>(&getContext());
   patterns.insert<ThreadwiseGemmRewritePattern>(&getContext());
+  populateAffineToStdConversionPatterns(patterns, &getContext());
+  populateLoopToStdConversionPatterns(patterns, &getContext());
```

Comment:
CMake rules would also have to be modified to tame shared lib build. Please refer to https://reviews.llvm.org/D79225 for an example.

Reply:
Thanks, lesson learned!

BTW, may I ask how did you noticed this? I am able to build all targets without the additional dependency in target_link_libaries, including `MLIRMIOpenTransforms`. Just curious.